internal/pkg/deploy/cloudformation/stack/transformers_test.go

```diff
@@ -1585,28 +1585,26 @@ func Test_convertSubscribe(t *testing.T) {
 		},
 		"subscription with empty topic subscriptions": {
 			inSubscribe: &manifest.SubscribeConfig{
-				Topics: []manifest.TopicSubscription{
+				Topics: &[]manifest.TopicSubscription{
 					{},
 				},
 			},
 			wantedError: fmt.Errorf(`invalid topic subscription "": %w`, errMissingPublishTopicField),
 		},
 		"valid subscribe": {
 			inSubscribe: &manifest.SubscribeConfig{
-				Topics: []manifest.TopicSubscription{
+				Topics: &[]manifest.TopicSubscription{
 					{
 						Name:    "name",
 						Service: "svc",
 					},
 				},
 				Queue: &manifest.SQSQueue{
```

Comment:
Can we add a test for an empty (or minimal) queue field in the manifest and make sure that the template object  gets populated with the default values appropriately?